### ISSUE TITLE
PRINT# basic example

### ISSUE THREAD

**Issue by MauriceMEGA65** [MEGA65/mega65-user-guide#164]
No error messages, but on screen nothing seems to happen.

see file PRINT#.ERR in BASIC.D81, send via Discord

**Comment by gurcei** [comment]
I downloaded the latest `BASIC65.D81` file from the filehost, but didn't see a `PRINT#.ERR` file within it.

Anyway, looking at this example, it wasn't designed to output anything to the screen, but more-so to write the `PRINT#` contents to a file called "TABLE".

In its present state, it didn't work for me, as it wanted to write the file to drive-9 (which I didn't have), so I tweaked this line to point to drive 8 instead:

FROM: `10 DOPEN#2,"TABLE",W,U9`
TO: `10 DOPEN#2,"TABLE",W,U8`

After I run the program now, I can then do a `DIR TA*` and observe that a new "TABLE" file has been created:
![image](https://user-images.githubusercontent.com/20013309/132285066-88fdf8b3-4f03-42b0-b957-d12a5a8b7652.png)

I can then view the contents of this SEQ file by typing `TYPE "TABLE"`:
![image](https://user-images.githubusercontent.com/20013309/132285165-69e9d152-2043-4653-a293-bbe0a17e6e58.png)

I'll aim to clarify the example's intentions in the document, and suggest the user to perform steps like I've mentioned here, to confirm the result of the program.
